File: frontend/hide_and_seek/src/app/models/gameResult.ts
export class GameResult {
  payoffMatrix: number[][];
  gameValue: number;
  computerProb: number[];
  pickedSpot: number;
  roundPayoff: number;

  constructor() {
    this.payoffMatrix = [];
    this.gameValue = 0;
    this.computerProb = [];
    this.pickedSpot = -1;
    this.roundPayoff = 0;
  }
}

File: backend/hide-and-seek/src/main/java/hns/model/GameResult.java
package hns.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;
@Data
@AllArgsConstructor
public class GameResult {
    List<List<Double>> payoffMatrix;
    double gameValue;
    List<Double> computerProb;
    int pickedSpot;
    double roundPayoff;
}